mui/notification.py

```diff
@@ -17,10 +23,15 @@ class UINotification(UIContextNotification):
     def __init__(self):
         UIContextNotification.__init__(self)
         UIContext.registerNotification(self)
+        self.mui_grpc_server_process: Optional[Popen] = None
 
     def __del__(self):
         UIContext.unregisterNotification(self)
 
+    def OnContextClose(self, context: UIContext) -> None:
+        if isinstance(self.mui_grpc_server_process, Popen):
+            self.mui_grpc_server_process.kill()
```

Comment:
I noticed that there seemed to be some zombie Z3 processes after stopping Manticore.

I tried changing this to `terminate()` but that didn't seem to stop the process 😞 

Have you noticed this? I assume the server isn't listening to our `terminate()` call and `kill()` seems to only apply to the Python server and not the other processes.